packages/core/src/components/pagination/pagination.css

```diff
@@ -86,7 +86,7 @@ we can not use this position for the actual button as it's in incorrect source o
 
 .hds-pagination__item-ellipsis {
   height: var(--link-height);
-  width: 24px;
+  width: 40px;
```

Comment:
This changes the outlook a bit. Is it a layout fix? 
I guess we have considered similar changes as a breaking change

Reply:
Not a breaking change since it is for the new component and fixes the layout

Reply:
We had a discussion about this and decided that we would not consider it a breaking change this time.